sklearn-sprint/guidelines.md

```diff
@@ -0,0 +1,83 @@
+
+![Hugging Face x Scikit-learn](./hfxsklearn.png)
+
+In this sprint, we will build interactive demos to scikit-learn documentation and later, contribute them. 
+
+## To get started 🤩
+
+1. Head to [this page](https://scikit-learn.org/stable/auto_examples/) and pick an example you’d like to build on. 
+2. Leave a comment to [this spreadsheet](https://docs.google.com/spreadsheets/d/14EThtIyF4KfpU99Fm2EW3Rz9t6SSEqDyzV4jmw3fjyI/edit?usp=sharing) with your name under Owner column, claiming the example. The spreadsheet has limited number of examples, feel free to add yours with a comment if it doesn’t exist in the spreadsheet.
+3. Start building!
+    
+    We will be hosting our applications in [scikit-learn](https://huggingface.co/scikit-learn) organization of Hugging Face. 
+    
+    For complete starters: in Hugging Face Hub, we have repositories for models, datasets, and [Spaces](https://huggingface.co/spaces). Spaces are special type of repositories hosting applications to try models easily. To write our apps, we will only be using Gradio. [Gradio](https://gradio.app/) is a library that lets you build a cool front-end application for your models, completely in Python. It supports many libraries, in this sprint, we will be using mostly visualization support (`matplotlib`, `plotly`, `altair` and more) and [skops](https://skops.readthedocs.io/en/stable/) integration (which you can launch an interface for a given classification or regression interface with one line of code). 
+    
+    In Gradio, there’s two ways to create a demo. One is to use `Interface`, which is a very simple abstraction. Let’s see a simple example.
+    
+    ```python
+    import gradio as gr
+    
+    def cancer_classifier(df):
+        # implement your classifier here
```

Comment:
Maybe show an example loading a joblib/pickle/skops file, otherwise people might think they need to implement a classifier from scratch?

(no sure, maybe just change the wording)

Reply:
I think it's redundant given we handle it in skops

Reply:
Maybe we could show how to load a model with skops here?

Reply:
But wouldn't it be confusing since there's already a gradio integration? I think as of now, it's best if no one loads anything, but rather assuming they already fit the model, they just call predict. maybe showing that is better

Reply:
I added inference instead

Reply:
SG!